src/minify.ts

```diff
@@ -97,7 +101,7 @@ export class ParcelCssMinifyPlugin {
         // Filter out already minimized
         !asset.info.minimized &&
         // Filter out by file type
-        CSS_FILE_REG.test(asset.name) &&
+        testRegExp.test(asset.name) &&
```

Comment:
```suggestion
        (testRegExp || CSS_FILE_REG).test(asset.name) &&
```

上面 `constructor` 里面的默认值逻辑不需要~